src/Honeycomb.OpenTelemetry/ConsoleLinksExporter.cs

```diff
@@ -0,0 +1,103 @@
+using OpenTelemetry;
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Honeycomb.OpenTelemetry
+{
+    /// <summary>
+    /// Writes links to the Honeycomb for all root spans
+    /// </summary>
+    public class ConsoleLinkExporter : BaseExporter<Activity>
+    {
+        private readonly HoneycombOptions _options;
+        private string _teamSlug;
+        private string _environmentSlug;
+
+        private bool IsEnabled = false;
+
+        /// <summary>
+        /// Initializes the <see cref="ConsoleLinkExporter" /> class
+        /// </summary>
+        /// <param name="options">Settings for Link generation</param>
+        public ConsoleLinkExporter(HoneycombOptions options)
+        {
+            _options = options;
+            InitTraceLinkParameters();
+        }
+
+        private void InitTraceLinkParameters()
+        {
+            if (string.IsNullOrEmpty(_options.ApiKey))
+                return;
+
+            var httpClient = new HttpClient();
+            httpClient.BaseAddress = new Uri("https://api.honeycomb.io/1/auth");
+            httpClient.DefaultRequestHeaders.Add("X-Honeycomb-Team", _options.ApiKey);
+
+            var response = httpClient.GetAsync("").GetAwaiter().GetResult();
+            if (!response.IsSuccessStatusCode) {
+                Console.WriteLine("Didn't get a valid response from Honeycomb");
+                return;
+            }

```

Comment:
We can use _options.TracesEndpoint instead of hardcoding.

```suggestion
            var httpClient = new HttpClient();
            httpClient.BaseAddress = new Uri(_options.TracesEndpoint);
            httpClient.DefaultRequestHeaders.Add("X-Honeycomb-Team", _options.ApiKey);

            var response = httpClient.GetAsync("1/auth").GetAwaiter().GetResult();
            if (!response.IsSuccessStatusCode) {
                Console.WriteLine("Didn't get a valid response from Honeycomb");
                return;
            }
```

Reply:
Given that someone can override the traces endpoint, is that an issue? My worry would be people overriding that, and not having the auth API... maybe I'm being a little over cautious?